pkg/controllers/edgeconnect/secret/secret.go

```diff
@@ -65,7 +65,18 @@ func PrepareConfigFile(ctx context.Context, instance *edgeconnectv1alpha1.EdgeCo
 	}
 
 	edgeConnectYaml, err := yaml.Marshal(cfg)
-	log.Debug(string(edgeConnectYaml))
+	log.Debug(safeEdgeConnectCfg(cfg))
 
 	return edgeConnectYaml, err
 }
+
+// Replace client secret with stars for debug logs
+func safeEdgeConnectCfg(cfg config.EdgeConnect) string {
```

Comment:
```suggestion
func sanitizeEdgeConnectCfg(cfg config.EdgeConnect) string {
```
But it's just an idea, not married to it.

Reply:
I'll merge and we can rename later.